components/sankey/data-bind.md

```diff
@@ -49,10 +49,10 @@ The `SankeyDataNode` object contains all the information for the node and its la
 | `Id` | `object` | The ID of the node. The ID is used to connect the nodes with the links. Required. |
 | `Label` | [`SankeyDataNodeLabel`](/blazor-ui/api/Telerik.Blazor.Components.SankeyDataNodeLabel) | Contains all the information for the node label - text, alignment, color, border and more. |
 | `Offset` | [`SankeyDataNodeOffset`](/blazor-ui/api/Telerik.Blazor.Components.SankeyDataNodeOffset)| The left and top offset of the node from the `<div class="k-sankey">` container. |
-| `Opacity` | `double?` | The opacity of the node. |
+| `Opacity` | `double?` <br/> (`1`) | The opacity of the node. |
 | `Padding` | `double?` | The minimum vertical space between two nodes. |
-| `Width` | `double?` | The width of the node. |
-| `Align` | `SankeyNodesAlign?` enum <br/> (`SankeyNodesAlign.Stretch`) | The alignment of the node. Supports the following values - `Stretch`, `Left`, and `Right`.|
+| `Width` | `double?` <br/> (`24`)| The width of the node. |
+| `Align` | [`SankeyNodesAlign?` enum](/blazor-ui/api/telerik.blazor.sankeynodesalign) <br/> (`SankeyNodesAlign.Stretch`) | The alignment of the node.|
```

Comment:
The enum type is known, so you can skip it to shorten the value (and the column width).

```suggestion
| `Align` | [`SankeyNodesAlign?` enum](/blazor-ui/api/telerik.blazor.sankeynodesalign) <br/> (`Stretch`) | The alignment of the node.|
```